charts/fluentd/values.yaml

```diff
@@ -150,6 +150,8 @@ metrics:
 dashboards:
   enabled: "true"
   namespace: ""
+  labels:
+    grafana_dashboard: '"1"'
```

Comment:
This doesn't need double escaping if you're quoting your labels.

```suggestion
    grafana_dashboard: "1"
```

Reply:
This is required because otherwise the "1" will disappear when the template is converted. The "1" is required as the label expects a string and not an int

Reply:
@dioguerra it's not required with the `{{ $val | quote }}` pattern that I've suggested above.